prototypes/entities.lua

```diff
@@ -117,7 +117,10 @@ local function add_recurrent_params(craft)
   craft.dying_explosion = "medium-explosion"
   craft.terrain_friction_modifier = 0
   craft.collision_box = {{-0.9, -1.3}, {0.9, 1.3}}
-  craft.collision_mask = {}
+  -- Original

```

Comment:
We should remove the "Original" comments, as the git commit has indicated the changes.